conformance/tests/httproute-response-header-modifier.go

```diff
@@ -27,7 +27,22 @@ import (
 )
 
 func init() {
-	ConformanceTests = append(ConformanceTests, HTTPRouteResponseHeaderModifier)
+	ConformanceTests = append(ConformanceTests,
+		HTTPRouteResponseHeaderModifier,
+		HTTPRouteBackendResponseHeaderModifier,
+	)
+}
+
+var HTTPRouteBackendResponseHeaderModifier = suite.ConformanceTest{
+	ShortName:   "HTTPRouteBackendResponseHeaderModifier",
+	Description: "An HTTPRoute backendRef has response header modifier filters applied correctly",
+	Features: []suite.SupportedFeature{
+		suite.SupportGateway,
+		suite.SupportHTTPRoute,
+		suite.SupportHTTPRouteBackendResponseHeaderModification,
+	},
+	Manifests: []string{"tests/httproute-response-header-modifier-backend.yaml"},
+	Test:      HTTPRouteResponseHeaderModifier.Test,
```

Comment:
Sorry I might miss the whole thing, if the tests are exactly the same, why do we need another supportedFeature ?